app/components/frame-viewer.cjsx

```diff
@@ -193,23 +195,24 @@ module.exports = React.createClass
     unless @state.panEnabled
       @setState panEnabled: true, =>
         this.refs.subjectImage.focus()
-  
-  togglePanOff: ->
-    @setState panEnabled: false
-  
+
+  togglePanOff: (e) ->
+    unless e.relatedTarget.title is "zoom out" or e.relatedTarget.title is "zoom in"
```

Comment:
When `e.relatedTarget` is null, the interface becomes stuck in pan mode when I try to switch back to annotate mode.